spec/javascripts/search_spec.js

```diff
@@ -1,97 +1,94 @@
-        });
+            })
+        })
 
         it('fires an event when trackSearchResultsAndSuggestions is called', function () {
-          var extractedURLs = GOVUK.search.extractSearchURLs($resultsList);
-          var $searchResults = $('#results .results-list');
-          GOVUK.search.trackSearchResultsAndSuggestions($searchResults);
-          expect(GOVUK.analytics.trackEvent).toHaveBeenCalled();
-        });
-      });
-    });
+          var $searchResults = $('#results .results-list')
+          GOVUK.search.trackSearchResultsAndSuggestions($searchResults)
+          expect(GOVUK.analytics.trackEvent).toHaveBeenCalled()
+        })
+      })
+    })
 
     describe('simple search results list', function () {
       beforeEach(function () {
-        GOVUK.analytics = GOVUK.analytics || {};
-        GOVUK.analytics.trackEvent = GOVUK.analytics.trackEvent || function(args) {};
-        spyOn(GOVUK.analytics, 'trackEvent');
-        $resultsList = $('<ol class="results-list"><li><h3><a href="guidance/content-design/what-is-content-design">Content design: planning, writing and managing content: What is content design?</a></h3></li><li><h3><a href="guidance/content-design/research-and-evidence">Content design: planning, writing and managing content: Research and evidence</a></h3><p>Tools and evidence to back up content design decisions.</p></li></ol>');
-        $results.append($resultsList);
-      });
+        GOVUK.analytics = GOVUK.analytics || {}
+        GOVUK.analytics.trackEvent = GOVUK.analytics.trackEvent || function (args) {}
+        spyOn(GOVUK.analytics, 'trackEvent')
+        $resultsList = $('<ol class="results-list"><li><h3><a href="guidance/content-design/what-is-content-design">Content design: planning, writing and managing content: What is content design?</a></h3></li><li><h3><a href="guidance/content-design/research-and-evidence">Content design: planning, writing and managing content: Research and evidence</a></h3><p>Tools and evidence to back up content design decisions.</p></li></ol>')
+        $results.append($resultsList)
+      })
 
       afterEach(function () {
-        $resultsList.remove();
-      });
+        $resultsList.remove()
+      })
 
       it('extracts all search result URLs', function () {
-        var extractedURLs = GOVUK.search.extractSearchURLs($resultsList);
+        var extractedURLs = GOVUK.search.extractSearchURLs($resultsList)
 
-        expect(extractedURLs.length).toEqual(2);
+        expect(extractedURLs.length).toEqual(2)
         expect(extractedURLs[0]).toEqual({
           href: 'guidance/content-design/what-is-content-design'
-        });
+        })
         expect(extractedURLs[1]).toEqual({
           href: 'guidance/content-design/research-and-evidence'
-        });
-      });
+        })
+      })
 
       it('fires an event when trackSearchResultsAndSuggestions is called', function () {
-        var extractedURLs = GOVUK.search.extractSearchURLs($resultsList);
```

Comment:
Looks like `extracedURLs` has been removed here, doesn't look like it was used so that's fine, just wanted to mention it as you'd specifically noted the removal of another unused var, but not this one.

Reply:
Yep, I removed this as it wasn't being used, and it didn't fail the tests. I'll edit the PR description to mention this too.